src/_path.h

```diff
@@ -1147,7 +1147,7 @@ bool __convert_to_string(PathIterator &path,
     double last_x = 0.0;
     double last_y = 0.0;
 
-    int size = 0;
+    size_t size = 0;
```

Comment:
This variable is only used in the `if(code<5)`, I think, so the definition can be moved there?